applications/PfemFluidDynamicsApplication/custom_processes/recover_volume_losses_process.hpp

```diff
@@ -212,7 +215,6 @@ namespace Kratos
 	  {
 	    if(i_node->Is(BOUNDARY) && i_node->IsNot(RIGID)){
 	      array_1d<double, 3>  Normal(3,0.0);
-	      Normal.clear();	 
 	      Normal    =i_node->FastGetSolutionStepValue(NORMAL);
```

Comment:
Wouldn't it be better to use 'array_1d<double, 3>&  Normal=i_node->FastGetSolutionStepValue(NORMAL);'?